src/core/lib/parse-shortcode.ts

```diff
@@ -22,7 +22,7 @@ export function isBlockShortcode(content: string) {
 
 function parseShortcodeCapture(capture: string): Shortcode | undefined {
   // match shortcode name
-  const nameMatch = capture.match(/^[a-zA-Z0-9_]+/);
+  const nameMatch = capture.match(/^\/?[a-zA-Z0-9_]+/);
```

Comment:
This does means we now support a name starting with `/` (e.g `{{< /name >}}`), right ? 

This part of the code is not just about escaping shortcode so I am wondering.

Reply:
> This does means we now support a name starting with / (e.g {{< /name >}}), right ?

That's right. As far as I can tell, other non-alphanumeric characters besides `/` aren't valid first characters fo Hugo shortcodes (although I didn't actually find a spec for them)

Reply:
Yes I think `/` is the only one at least obvious, for closing shortcode. 

Probably allowing this for our shortcode name won't be that bad, especially if not documented, no one will build a shortcode with leading `/` 😅 

> One of the comment in the PR is mentionning, teaching about unsupported shortcode for example, and this fix only won't solve it.

Still about the issue, this PR does not solve this then. But I believe we can document the limitation as `{{{< ... >}}}` escaping works; So it seems ook.

Reply:
Wait. I'm sorry if I'm being stupid again, but does the smoke test I added not work for you?

Reply:
No no all good on your side. I am just pushing on some edge case. As shared above, I was thinking of some tutorial document where you would show an unsupported shortcode syntax like below. 

````
```{shortcodes=false}
# Shortcode syntax
{{< test something >}}
# don't start with weird characters
{{< ~test something >}}
```
````

As a user, reading the doc, you would expect `shortcodes=false` to be escaping the shortcodes, so not 'processing' them. But this would throw an error. 

Anyhow, I understand now better how we handle shortcodes. 

Conclusion is: 

* `{{{< ... >}}}` works for escaping any shortcode name (valid or not valid)
* `` ```{shortcodes=false} ``  will only works on valid shortcode name, because we are parsing and check shortcode validity anyway. 

So as I said, this is a edge case. 

This PR can be merge to support closing shortcode syntax from hugo.